.travis.yml

```diff
@@ -1,10 +1,10 @@
 language: ruby
-sudo: false
 cache: bundler
 rvm:
-  - 2.4.6
-  - 2.5.5
-  - 2.6.3
+  - 2.4
+  - 2.5
+  - 2.6
+  - 2.7
```

Comment:
Simplifies .travis.yml and also avoids "commit-noise" when patch versions get released. Of course this is up to preference, so see this as a proposition.